packages/interaction-kit/src/cli/dev.ts

```diff
@@ -37,65 +75,54 @@ export default async function dev(argv?: string[]) {
 	);
 
 	const port = args["--port"] ?? 3000;
-	let child: ChildProcess | null = null;
 
-	// Listen for config file changes and let user know they need to reload
-	const configWatcher = chokidar.watch(CONFIG_FILES, {
-		ignoreInitial: true,
-	});
-	configWatcher.on("change", (path) => {
-		console.log(
-			`Change detected in ${path} - please restart your application!`
-		);
-	});
+	// eslint-disable @typescript-eslint/no-unused-expressions
+	// @ts-expect-error We're faking worker state
+	global.APPLICATION_ID = process.env.APPLICATION_ID;
+	// @ts-expect-error We're faking worker state
+	global.PUBLIC_KEY = process.env.PUBLIC_KEY;
+	// @ts-expect-error We're faking worker state
+	global.TOKEN = process.env.TOKEN;
+	// eslint-enable @typescript-eslint/no-unused-expressions
 
-	// Watch for changes requiring application reloads
-	const botWatcher = chokidar.watch(BOT_FILES, {
-		// ignoreInitial: true,
-	});
-	const handler = async () => {
-		console.log("Reloading application");
-
-		child?.kill();
-		child = spawn("ikit", ["server", "-p", port.toString()], {
-			stdio: "inherit",
-		});
-	};
-
-	botWatcher.on("change", (path) => {
-		console.log(`${path} changed, reloading`);
-		void handler();
-	});
-	botWatcher.on("add", (path) => {
-		console.log(`${path} was added, reloading`);
-		console.log(path);
-		void handler();
-	});
-	botWatcher.on("unlink", (path) => {
-		console.log(`${path} was removed, reloading`);
-		void handler();
-	});
+	// const server = new Miniflare({
```

Comment:
why is this commented out?

Reply:
It doesn't work with the current system; leaving it commented because it should match up with the APIs for the new wrangler 2 API